src/solver/utils/ortools_utils.cpp

```diff
@@ -99,27 +99,24 @@ void ProblemSimplexeNommeConverter::CopyMatrix(const MPSolver* solver)
     }
 }
 
-void ProblemSimplexeNommeConverter::UpdateCoefficient(unsigned idxVar,
-                                                      MPSolver* solver,
-                                                      MPObjective* const objective)
+void ProblemSimplexeNommeConverter::CreateVariable(unsigned idxVar,
+                                                   MPSolver* solver,
+                                                   MPObjective* const objective)
 {
-    double min_l = 0.0;
-    if (problemeSimplexe_->Xmin != NULL) // TODO[FOM] Remove enclosing if ?
-    {
-        min_l = problemeSimplexe_->Xmin[idxVar];
-    }
+    double min_l = problemeSimplexe_->Xmin[idxVar];
     double max_l = problemeSimplexe_->Xmax[idxVar];
-    const MPVariable* var = solver->MakeNumVar(min_l, max_l, variableNameManager_.GetName(idxVar));
+    bool intVar = problemeSimplexe_->IntegerVariable(idxVar);
+    const MPVariable* var = solver->MakeVar(min_l, max_l, intVar, variableNameManager_.GetName(idxVar));
```

Comment:
The main change is here, we use `MPSolver::MakeVar` instead of `MPSolver::MakeNumVar`.